diff --git a/node_modules/jest-snapshot/build/index.d.ts b/node_modules/jest-snapshot/build/index.d.ts
index 3f20807..b47c90a 100644
--- a/node_modules/jest-snapshot/build/index.d.ts
+++ b/node_modules/jest-snapshot/build/index.d.ts
@@ -148,7 +148,7 @@ export declare class SnapshotState {
   fail(testName: string, _received: unknown, key?: string): string;
 }
 
-declare type SnapshotStateOptions = {
+export declare type SnapshotStateOptions = {
   updateSnapshot: Config.SnapshotUpdateState;
   prettierPath: string;
   expand?: boolean;
