See https://github.com/vitest-dev/vitest/issues/3704

diff --git a/node_modules/vite-node/dist/server.d.ts b/node_modules/vite-node/dist/server.d.ts
index a21e596..80b6bd2 100644
--- a/node_modules/vite-node/dist/server.d.ts
+++ b/node_modules/vite-node/dist/server.d.ts
@@ -1,3 +1,4 @@
+//@ts-expect-error
 import { TransformResult, ViteDevServer } from 'vite';
 import { E as EncodedSourceMap } from './types.d-7442d07f.js';
 import { g as DebuggerOptions, D as DepsHandlingOptions, f as ViteNodeServerOptions, F as FetchResult, e as ViteNodeResolveId } from './types-e8623e9c.js';
diff --git a/node_modules/vite-node/dist/types-e8623e9c.d.ts b/node_modules/vite-node/dist/types-e8623e9c.d.ts
index dfe926f..ac99fe8 100644
--- a/node_modules/vite-node/dist/types-e8623e9c.d.ts
+++ b/node_modules/vite-node/dist/types-e8623e9c.d.ts
@@ -1,3 +1,4 @@
+//@ts-expect-error
 import { ViteHotContext } from 'vite/types/hot';
 import { E as EncodedSourceMap } from './types.d-7442d07f.js';

